tests/test_api_methods/test_administrative_local.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+from unittest.mock import patch, Mock
+
+from fake_ubersmith.api.administrative_local import AdministrativeLocal
+
+
+class TestAdministrativeLocal(unittest.TestCase):
+    def setUp(self):
+        self.admin = AdministrativeLocal()
+
+    def test_shutdown_works(self):
+        patch_path = 'fake_ubersmith.api.administrative_local.request'
+        environ_path = 'werkzeug.server.shutdown'
+
+        with patch(patch_path, Mock(environ={environ_path: Mock})):
+            self.admin.shutdown()
```

Comment:
Yeah it was a strange validation I had. But, from our offline conversation, I agree that the test would be more robust using the test_client. Will change it. Thanks.